src/cplscheme/CompositionalCouplingScheme.cpp

```diff
@@ -242,9 +241,10 @@ bool CompositionalCouplingScheme::isActionRequired(
 {
   PRECICE_TRACE(actionName);
   bool isRequired = false;
-  for (const Scheme &scheme : _couplingSchemes) {
-    if (not scheme.onHold) {
-      isRequired |= scheme.scheme->isActionRequired(actionName);
+  for (auto scheme : allSchemes()) {
+    if (scheme->isActionRequired(actionName)) {
+      isRequired = true;
+      break;
```

Comment:
`std::any_of`?

Reply:
This will be completely changed in #1487, so I rather leave it for now.